/**:
    ros__parameters:
        common:
            lid_topic:  "/livox/lidar"
            imu_topic:  "/livox/imu"
            time_sync_en: false         # ONLY turn on when external time synchronization is really not possible
            time_offset_lidar_to_imu: 0.0  

            filter_size_surf: 0.1
            # filter_size_map: 0.5
            # cube_len: 200.0

        kf:
            maximum_iter: 4
            epsi: 0.001
            NUM_MATCH_POINTS: 5
            plane_thr: 0.1
            
            degen_rate_max: 3.0
            degen_weight: 1.0

        preprocess:
            lidar_type: 1                # 1 for Livox serials LiDAR, 2 for Velodyne LiDAR, 3 for ouster LiDAR, 
            scan_line: 4
            blind: 2.0
            point_filter_num: 1

        mapping:
            acc_cov: 0.1
            gyr_cov: 0.1
            b_acc_cov: 0.0001
            b_gyr_cov: 0.0001
            det_range:     450.0
            extrinsic_est_en:  false      # true: enable the online estimation of IMU-LiDAR extrinsic
            extrinsic_T: [ -0.011, -0.02329, 0.04412 ]
            extrinsic_R: [ 1., 0., 0.,
                            0., 1., 0.,
                            0., 0., 1.]

        publish:
            path_en:  true
            scan_publish_en:  true       # false: close all the point cloud output
            dense_publish_en: true       # false: low down the points number in a global-frame point clouds scan.
            runtime_pos_log: true
 

        pcd_save:
            pcd_save_en: 1  # 0 1(one pcd)  2()
            filter_size_savemap: 0.1
            savemap_dir: /home/qianliyue/D:/RepoMy/save/lt/m3/
            
           
